Makefile

```diff
@@ -28,20 +28,29 @@ APP_MIME_SIG = application/x-vnd.Be-BTL.Toner
 #	means this Makefile will not work correctly if two source files with the
 #	same name (source.c or source.cpp) are included from different directories.
 #	Also note that spaces in folder names do not work well with this Makefile.
-SRCS = TonerApp.cpp TonerWindow.cpp TonePlayer.cpp TonerView.cpp
+SRCS = \
+	 TonePlayer.cpp  \
+	 TonerApp.cpp  \
+	 TonerWindow.cpp  \
+
 
 #	Specify the resource definition files to use. Full or relative paths can be
 #	used.
-RDEFS = Toner.rdef
+RDEFS = \
+	 Toner.rdef  \
```

Comment:
No reasonn to prepare for more lines here, so I suggest moving the rdef directly into the RDEFS line ybove

Reply:
I've never touched that part of the Makefile, maybe that's generated before.